Python-Home-Challenges/Python-home-challenge#01-MaorWeiss.py

```diff
@@ -0,0 +1,27 @@
+# HW03 - Open a text file and returns the most recurring word in that file.
+
+fp = open('/file.txt', 'r')
+# Open file #
+f = fp.readline()
+# Read file #
+split_words_from_file = f.split()
+
+the_most_recurring_word = ''
+biggest_counting = 0
+count = 0
+
+copy_list = split_words_from_file
```

Comment:
no reason to declare new variable here

Reply:
Do you mean new variables? or just count variable..? @AviadP

Reply:
@AviadP

Reply:
there is no reason just to assign "split_words_from_file" into "copy_list"